### ISSUE TITLE
File Version Identifier

### ISSUE THREAD

**Issue by fubuloubu** [aragon/radspec#32]
Add `@version` to top level radspec for the file

This might be a semvar string, or it could be a subset of the hash of the document's sha3. Or both! (e.g. `@version v0.1.0-1a2b3c4d`)

**Comment by sohkai** [comment]
> it could be a subset of the hash of the document's sha3

This would be technically impossible unless it was the subset of the document without the file identifier.

Similar to #33, I'm not sure this would be useful to users (although it should definitely be a developing best practice!).

**Comment by fubuloubu** [comment]
>> it could be a subset of the hash of the document's sha3
> This would be technically impossible unless it was the subset of the document without the file identifier.

Yeah, usually this would be accomplished with calculating the hash assuming 0 or empty in that location, then placing it there as a checksum. It could be validated that way.
